testsuite/src/test/java/org/ironjacamar/core/connectionmanager/pool/NoCredentialsTestCase.java

```diff
@@ -96,9 +98,13 @@ private ResourceAdapterArchive createResourceAdapter() throws Throwable
    @Deployment(order = 2)
    private ResourceAdaptersDescriptor createActivation() throws Throwable
    {
-      return ResourceAdapterFactory.createUnifiedSecurityDeployment(null, null);
+      return ResourceAdapterFactory.createUnifiedSecurityDeployment(null, null,
+            TransactionSupport.TransactionSupportLevel.NoTransaction,
+            "UnifiedSecurityConnectionFactory", "unified-security.rar");
    }
 
+
```

Comment:
We need a test case that shows that a transaction doesn't impact security and number of CLs